cargo-quickinstall/tests/integration_tests.rs

```diff
@@ -51,8 +62,16 @@ fn do_dry_run_for_nonexistent_package() {
         target: "unknown".to_string(),
     };
 
-    let result = do_dry_run_curl(&crate_details);
+    let result = do_dry_run_curl(&crate_details).unwrap();
 
-    let expected = r#""cargo" "install" "nonexisting_crate_12345" "--version" "99""#;
+    let expected = "cargo install nonexisting_crate_12345 --version 99";
     assert_eq!(expected, &result);
 }
+
+#[test]
+fn test_get_latest_version() {
+    assert_eq!(
+        get_latest_version("cargo-quickinstall").unwrap(),
+        env!("CARGO_PKG_VERSION")
```

Comment:
this will break `git bisect` I think: as soon as you check out an old version of the code, this test will start to fail.

Reply:
Fixed.
I've replaced this with a `git describe` to find the latest tag.